integration_tests/dbt_project.yml

```diff
@@ -1,14 +1,24 @@
 config-version: 2
 
 name: 'stripe_source_integration_tests'
-version: '0.12.0'
+version: '0.12.1'
 
 
 profile: 'integration_tests'
 
 vars:
   stripe_schema: stripe_source_integrations_tests_6
+  
   stripe_source:
+    card_pass_through_columns:
```

Comment:
@fivetran-jamie Should we remove this? I assume it was only for testing.

Reply:
Figured there's no harm in including them in integration tests moving forward 🤷